Server/src/server/dataCenter/DataCenter.java

```diff
@@ -1,6 +1,8 @@
 package server.dataCenter;
 
 import com.google.gson.GsonBuilder;
+
```

Comment:
Maybe this is one for @skeet70,  but should the Server code have anything do with the config?

With that said, when in local host the server needs to do some things with the AppData folder.  Is there a smarter way todo this?

Reply:
Not wrong, but the `Config` class is more like "configuration directory" which is `AppData/ourshit` for Windows `ApplicationSupport/ourshit` for Mac, and `.config/ourshit` for Linux. I agree that the server shouldn't care about the `config.properties` file, but it's fine for it to care about other stuff in that folder.